src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -62,6 +62,18 @@ Integer insertDataset(
       @Bind("dataUse") String dataUse,
       @Bind("dacId") Integer dacId);
 
+  @SqlUpdate("""
+            UPDATE dataset
+            SET name = :name,
+                update_date = :updateDate,
+                update_user_id = :updateUserId,
+                needs_approval = :needsApproval,
+                dac_id = :dacId
+            WHERE dataset_id = :datasetId
+            """)
+  Integer updateDatasetNew(@Bind("datasetName") String name, @Bind("updateDate") Timestamp updateDate, @Bind("updateUserId") Integer updateUserId,
```

Comment:
Nit, naming/duplication. Is there a reason we can't refactor the existing update?